### ISSUE TITLE
Man-made class means building

### ISSUE THREAD

**Issue by promesse** [weiyithu/SurroundOcc#25]
Thank you for you great job!
I'd like to ask, does man-made refer to buildings?

**Comment by weiyithu** [comment]
Yeah, I think you can treat it as buildings.

**Comment by promesse** [comment]
> Yeah, I think you can treat it as buildings.

Thanks for your reply！
I also want to ask, how can I load the trained model and continue training?

**Comment by weiyithu** [comment]
Hi, you can try [--resume-from](https://github.com/weiyithu/SurroundOcc/blob/a2fb7d58e5e0a921a819fc8903fc160fdabc15cd/tools/train.py#L38).

**Comment by promesse** [comment]
> Hi, you can try [--resume-from](https://github.com/weiyithu/SurroundOcc/blob/a2fb7d58e5e0a921a819fc8903fc160fdabc15cd/tools/train.py#L38).
Thank you for your lightning reply.
If I've trained 24 epochs and want to train more epochs, do I have to modify the total_epoches for example 50 and try --resume-from?
![image](https://user-images.githubusercontent.com/26239060/233559852-e37a57c4-01b6-4d5a-9d8f-51d6463ad9e5.png)

**Comment by weiyithu** [comment]
No, if you wthe epoch number is different with the epoch in config file, you should train from scratch since the lr policy is changed. Cosine learning rate is dependent on epoch number.

**Comment by promesse** [comment]
> 
Will it be better to continue training than to retrain?
![image](https://user-images.githubusercontent.com/26239060/233755331-081fda00-25e4-4768-ae2f-051e410f031e.png)

**Comment by weiyithu** [comment]
Sorry, I have not try this setting but I think retraining should be better.

**Comment by promesse** [comment]
> Sorry, I have not try this setting but I think retraining should be better.
Thank you very much for your advice
